### ISSUE TITLE
Reg\Register an Operation - Opted-In Operation Details in Operation edit/detail page

### ISSUE THREAD

**Issue by Sepehr-Sobhani** [bcgov/cas-registration#2094]
#### Dependencies

- Blocked by #1952 and #1879 
- Blocking

#### Acceptance Criteria:

Given an operation has registered as an "Opted-in Operation"
When an authenticated external user with access to that operation navigates to view their Operation details
Then they see [this](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18025-1179&t=uNuS5PhP3MMoN80I-4)
And they have the option to [edit](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18025-1336&t=uNuS5PhP3MMoN80I-4) and update the fields contained in the Opt-in form

#### Development Checklist:

- [x] Update the operation get endpoint to include opted-in information
- [x] Update the operation schema to display opted-in data if the operation is an opted-in operation.
- [x] Add additional tests to the front-end and back-end to ensure that displaying and updating opted-in information is fully covered.

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner
